frontend/src/composables/use-collection-meta.ts

```diff
@@ -0,0 +1,57 @@
+import { computed } from "vue"
+
+import { TranslateResult } from "vue-i18n"
+
+import type { SupportedMediaType } from "~/constants/media"
+import type { CollectionParams } from "~/types/search"
+import { useI18n } from "~/composables/use-i18n"
+import { useProviderStore } from "~/stores/provider"
+
+import type { ComputedRef } from "vue"
+
+export const useCollectionMeta = ({
+  collectionParams,
+  mediaType,
+  i18n,
+}: {
+  collectionParams: ComputedRef<CollectionParams | null>
+  mediaType: SupportedMediaType
+  i18n: ReturnType<typeof useI18n>
+}) => {
+  const pageTitle = computed(() => {
+    // The default page title. It should be overwritten by the specific collection type.
+    let title: string | TranslateResult =
+      "Openly Licensed Images, Audio and More"
```

Comment:
Is this used as a fallback just to appease TypeScript? If so, we can use a return in each case instead of assigning to a variable. Otherwise, I suppose this should be translated too, in case for some reason it is used? Not-blocking, but definitely needs an issue to fast-follow.

Reply:
Refactored to not use `string | TranslateResult`.

I tried using the `switch/case`, but because the function is inside a computed, it causes an `Expected to return a value in computed function  vue/return-in-computed-property` linting error, even though all of the `collectionParams.collection` options are exhausted. That's why I used a redundant if cause.